src/api/java/com/enderio/api/io/IIOConfig.java

```diff
@@ -56,4 +56,9 @@ default void cycleMode(Direction side) {
      */
     @UseOnly(LogicalSide.CLIENT)
     boolean renderOverlay();
+
+    /**
+     * Whether the IO config is unchanged from initial values.
+     */
+    boolean isDefault();
```

Comment:
Personally not a fan of having this method, but I'm unsure of a way that would alleviate the need for it